.github/workflows/auto-copilot-review.yml

```
@@ -0,0 +1,124 @@
+name: Auto Copilot Review
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)